src/views/Alteration.vue

```diff
@@ -39,7 +42,11 @@
       </section>
 
       <!-- FUTURE: set `pleaseValidate` when user clicks File and Pay -->
-      <alteration-summary class="mt-10" :pleaseValidate="true" />
+      <alteration-summary
+        class="mt-10"
+        :pleaseValidate="true"
```

Comment:
Politeness 👍

Reply:
Hehe, this will need to be refactored anyway (future ticket).